tools/targets/__init__.py

```diff
@@ -1,6 +1,6 @@
 """
 mbed SDK
-Copyright (c) 2011-2020 ARM Limited
+Copyright (c) 2011-2021 ARM Limited
```

Comment:
why do we need this file (changing it here) ? I would expect to not to use this one at all. It's one common that we should not need. If we need something from here, we should move it to targets scripts rather. This file contains too many information for the old build system.

Reply:
This file is not used for the new build system. It was modified because it uses one of the API that was modified. However, I can add a wrapper API function so this stays as it was.

Reply:
Done. Changes to this file have been reverted.